subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/JavaCompile.java

```diff
@@ -219,6 +221,12 @@ private DefaultJavaCompileSpec createSpec() {
         spec.setTargetCompatibility(getTargetCompatibility());
         spec.setSourceCompatibility(getSourceCompatibility());
         spec.setCompileOptions(compileOptions);
+        if (compileOptions.isIncrementalAnnotationProcessing()) {
+            File incrementalAnnotationProcessorWorkingDir
+                    = new File(getProject().getBuildDir(), "intermediates/annotationProcessing");
```

Comment:
⭕️ We don't use the term "intermediates". You can use "tmp" for this.

Reply:
Fixed.